images/basic/00-Topology-v1.2.0.MSZ-demo.svg

```diff
@@ -1,3 +1,3 @@
9 104.68 179.84 98.9 177.2 C 100.21 170.04 97.73 162.7 92.34 157.83 C 86.71 152.84 78.43 152.34 72.24 156.62 C 69.28 147.65 60.31 142.4 51.37 144.41 C 42.17 147.5 36.32 156.59 37.28 166.31 C 30.85 163.63 23.46 166.48 20.45 172.8 C 16.21 172.15 11.92 173.43 8.55 176.35 C 5.17 179.27 3 183.59 2.53 188.31 C 1.58 197.14 7.29 205.29 15.86 207.35 Z" fill="#7d7d7d" stroke="none" pointer-events="all"/><path d="M 120.17 172.92 L 187.83 108.08" fill="none" stroke="#9e958e" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><ellipse cx="118" cy="175" rx="3" ry="3" fill="#9e958e" stroke="#9e958e" pointer-events="all"/><ellipse cx="190" cy="106" rx="3" ry="3" fill="#9e958e" stroke="#9e958e" pointer-events="all"/><path d="M 123.7 195.85 L 183 195.99" fill="none" stroke="#9e958e" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="6 6" pointer-events="stroke"/><ellipse cx="120.7" cy="195.84" rx="3" ry="3" fill="#9e958e" stroke="#9e958e" stroke-width="2" pointer-events="all"/><ellipse cx="186" cy="196" rx="3" ry="3" fill="#9e958e" stroke="#9e958e" stroke-width="2" pointer-events="all"/><image x="344.5" y="195.5" width="34" height="34" xlink:href="data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIGlkPSJTZWN1cml0eV9MaXN0cyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiIHZpZXdCb3g9IjAgMCAxNjEuOSAxNjIiIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDE2MS45IDE2MiIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+JiN4YTs8Zz4mI3hhOwk8cGF0aCBmaWxsPSIjMzEyRDJBIiBkPSJNMTI3LDgzVjQwLjNsLTQ2LTE5LjdsLTQ2LDE5LjhsMC4yLDQ1LjFsMC4xLDAuNGM2LjgsMzkuNSw0Mi42LDU0LjEsNDQuMSw1NC43bDEuOSwwLjhsMS44LTAuOSAgIEMxMjYuOSwxMTcuOSwxMjcsODQuNCwxMjcsODN6IE0xMTgsODMuMWMwLDEuMi0wLjMsMjguNi0zNy4xLDQ4LjNjLTYuOS0zLjQtMzEuNS0xNy4zLTM2LjctNDYuN2wtMC4xLTM4LjRsMzctMTUuOWwzNywxNS45VjgzLjF6Ii8+JiN4YTsJPHBvbHlnb24gZmlsbD0iIzMxMkQyQSIgcG9pbnRzPSI5OS41LDY5LjYgOTUuMSw3NCA5MC43LDY5LjYgODcuNyw3Mi43IDkyLjEsNzcuMSA4Ny43LDgxLjQgOTAuNyw4NC41IDk1LjEsODAuMSA5OS41LDg0LjUgICAgMTAyLjYsODEuNCA5OC4yLDc3LjEgMTAyLjYsNzIuNyAgIi8+JiN4YTsJPHBvbHlnb24gZmlsbD0iIzMxMkQyQSIgcG9pbnRzPSIxMDQuMSw1Mi45IDEwMS4xLDQ5LjggOTIuNiw1OCA4OS41LDU0LjMgODYuMiw1Ny4xIDkyLjMsNjQuMyAgIi8+JiN4YTsJPHBvbHlnb24gZmlsbD0iIzMxMkQyQSIgcG9pbnRzPSIxMDQuMSw5MiAxMDEuMSw4OC45IDkyLjYsOTcuMSA4OS41LDkzLjQgODYuMiw5Ni4yIDkyLjMsMTAzLjQgICIvPiYjeGE7CTxyZWN0IHg9IjU3LjgiIHk9Ijc0LjUiIGZpbGw9IiMzMTJEMkEiIHdpZHRoPSIyMS4yIiBoZWlnaHQ9IjYuNiIvPiYjeGE7CTxyZWN0IHg9IjU3LjgiIHk9Ijk0LjEiIGZpbGw9IiMzMTJEMkEiIHdpZHRoPSIyMS4yIiBoZWlnaHQ9IjYuNiIvPiYjeGE7CTxyZWN0IHg9IjU3LjgiIHk9IjU0LjkiIGZpbGw9IiMzMTJEMkEiIHdpZHRoPSIyMS4yIiBoZWlnaHQ9IjYuNiIvPiYjeGE7PC9nPiYjeGE7PC9zdmc+" preserveAspectRatio="none"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
What changed?

Reply:
not sure if this was an accidental check-in - your updated picture has what I did.  Maybe not the cert part.